prep-javascript-arrays/main.js

```diff
@@ -0,0 +1,14 @@
+const languages = ["html", "css", "javascript", "react", "node", "postgress"]
+console.log(languages)
+
+var firstElement = languages[0];
+console.log(firstElement);
+
+var thirdElement = languages[2];
+console.log(thirdElement);
+
+var length = languages.length;
+console.log(length);
+
+var lastElement = languages [5];
```

Comment:
Here we want to use the `length` property to get the last item in the array. While using `5` works in this exact instance as soon as the array size changes it will no longer be the last element so we want to use something that will work every time regardless if the array changes size or not.

We use the length of the array minus 1 to calculate the last index in an array.